anitya/templates/logs.html

```diff
@@ -94,11 +94,11 @@
 
 </div>
 
-
-<table>
+<br>
```

Comment:
Ah ok, cool :)

Change looks good I just wanted to test it locally before merging it :)